apps/vor/pkg/student/student.go

```diff
@@ -54,7 +55,65 @@ func authorizationMiddleware(s rest.Server, store Store) func(next http.Handler)
 		})
 	}
 }
+func registerClass(s rest.Server, store Store) http.Handler {
+	type responseBody struct {
+		Id          string     `json:"id"`
+		Name        string     `json:"name"`
+		DateOfBirth *time.Time `json:"dateOfBirth,omitempty"`
+	}
+	return s.NewHandler(func(w http.ResponseWriter, r *http.Request) *rest.Error {
+		id := chi.URLParam(r, "studentId")
 
+		student, err := store.Get(id)
+		if err != nil {
+			return &rest.Error{http.StatusNotFound, "Can't find student with specified id", err}
+		}
+
+		response := responseBody{
+			Id:          student.Id,
+			Name:        student.Name,
+			DateOfBirth: student.DateOfBirth,
+		}
+		if err := rest.WriteJson(w, response); err != nil {
+			return rest.NewWriteJsonError(err)
+		}
+		return nil
+	})
+}
+func registerAttendance(s rest.Server, store Store) http.Handler {
+	type requestBody struct {
```

Comment:
We should accept the date of the attendance from the client so that the user can register student attendance for any date.

Reply:
So instead of auto inserting the date, we let the user choose the date? It means user can add attendance for previous days?

Reply:
Yes, users should be able to edit past attendance too.